src/state/utils.ts

```diff
@@ -41,3 +42,23 @@ export function safeStoredLocation(): LocationResponse | undefined {
 export function randomSeed(): number {
   return Math.floor(Math.random() * 1000);
 }
+
+const { Point } = BaseBuilder.MODELS;
+export function convertCoordinatesToGPX(coordinates: LatLong[]): string {
+  const pointArray = coordinates.map((value) => new Point(value[0], value[1]));
```

Comment:
You could also deconstruct value as `([lat, lng] => new Point(lat, lng))` for extra coolness